docker/Dockerfile

```diff
@@ -1,37 +1,16 @@
 #
-# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+# Copyright 2023, Proofcraft Pty Ltd
```

Comment:
If this is based on another file somewhere else, it's worth mentioning where it comes from in the commit message (in case in the future this needs to be updated the same way as the source file).

Reply:
> This is from scratch, I haven't based it on anything else.

Considering this work is part of the web doc revamp project, shouldn't the copyright be different? Or can I also just copyright new files in my name as long as it's open source? I assumed I had to assign everything to the seL4 foundation.

Reply:
Yes, it's Ok to keep copyright to yourself, the only requirements are the DCO and using the foundation-approved licenses. No need to assign copyright to the foundation.